src/main/java/RacingCar.java

```diff
@@ -0,0 +1,113 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingCar {
+
+    static String move = "-";
+
+    static List<Car> recordMovingCar = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        InputView userInput = scannerInput();
+
+        int numberOfCarsValue = userInput.getNumberOfCars();
+        int numberOfAttemptsValue = userInput.getNumberOfAttempts();
+
+        System.out.println("======= READY =========");
+        for (int carNumber=0; carNumber < numberOfCarsValue; carNumber++) { System.out.println(move); }
+
+        System.out.println("======= START!! =======");
+        randomMove(numberOfCarsValue, numberOfAttemptsValue);
+
+    }
```

Comment:
```suggestion
    public static void main(String[] args) {
        int numberOfCars = InputView.inputNumberOfCars();
        int numberOfAttempts = InputView.inputNumberOfAttempts();
        RacingGame racingGame = new RacingGame(numberOfCars);
        GameResult result = racingGame.play(numberOfAttempts);
        ResultView.draw(result);
    }
```

- 한번 위와 같이 main 메소드를 작성하고 진행해보시겠어요? (없는 객체들은 추가해서 진행해보세요.)